### DISCUSSION TITLE
About @remix-run/dev bundleAnalyze

### DISCUSSION THREAD

**Discussion by 19Qingfeng** [remix-run/remix#6416] [Proposals]
Whether you can consider supporting the metafile option at the @remix-run/dev build time.

I believe that most people will be concerned about the size of their applications, and so will I.But I can't find any option to enter metafile on remix.config.js.

I checked the source code on the @remix-run/dev and found that metafile has indeed been generated inside esbuild and remix use that to do something, and whether we can consider exposing it through configuration. So that I can be based on [Esbuild Analyze](https://esbuild.github.io/analyze/) to analyze my web project.

**Comment by kiliman** [comment]
I have a patch to generate the analysis file. I should probably create a PR. 

https://github.com/kiliman/remix-esbuild-analysis

**Reply by 19Qingfeng** [reply]
> I have a patch to generate the analysis file. I should probably create a PR.
> 
> https://github.com/kiliman/remix-esbuild-analysis

okay, thank you.

**Comment by MichaelDeBoey** [comment]
Remix now outputs metafiles for bundle analyses as of `1.19.0`

https://github.com/remix-run/remix/releases/tag/remix%401.19.0
